packages/cashier/src/pages/deposit/deposit.tsx

```diff
@@ -120,7 +120,7 @@ const Deposit = observer(({ setSideNotes }: TDeposit) => {
         return <CryptoTransactionsHistory />;
     }
 
-    if (is_deposit || is_eu) {
+    if (is_deposit) {
```

Comment:
we have the `show_eu_related_content` flag instead `is_eu`  for DIEL project , use this
```suggestion
    if (is_deposit || show_eu_related_content) {
```

Reply:
> show_eu_related_content

these varibles ('is_eu' and 'show_eu_related_content') are both from 'traders_hub' store and its both are true when flag EU choosed, but my task is cashier for EU and Non-EU looks the same (no error and no crypto_deposit if the client didn't make deposit before).